specs/agents/handling-huge-traces/tracing-spans-drop-fast-exit.md

```diff
@@ -22,7 +22,7 @@ Additionally, spans that lead to an error can't be discarded.
 |                |            |
 |----------------|------------|
 | Type           | `duration` |
-| Default        | `1ms`      |
+| Default        | `0ms`      |
 | Central config | `true`     |
 
 The minimum allowed duration for this setting is `1us` (microsecond). Agents may need to
```

Comment:
It seems the new default contradicts this requirement.

Reply:
This is handled in https://github.com/elastic/apm/pull/607